ecosystem/sep-0010.md

```diff
@@ -136,8 +139,7 @@ To validate the challenge transaction the following steps are performed by the s
 * verify that transaction source account is equal to the server's signing key;
 * verify that transaction has time bounds set, and that current time is between the minimum and maximum bounds;
 * verify that transaction contains a single [Manage Data](https://www.stellar.org/developers/guides/concepts/list-of-operations.html#manage-data) operation that:
-  * has a non-null source account 
-  * has the service's home domain as the key of the operation
+  * has a non-null source account
```

Comment:
Same point as above, about being explicit about what shouldn't be validated.

Reply:
I think if we add this statement we can leave this section as is: https://github.com/stellar/stellar-protocol/pull/740#discussion_r502068312

Also FYI this section is actually what the server must do to validate the challenge transaction passed back.